.github/workflows/release-post-merge.yml

```diff
@@ -83,17 +83,27 @@ jobs:
           make release-helm JWT_RSA_PEM_KEY_BASE64="${{ secrets.AKO_RELEASER_RSA_KEY_BASE64 }}" \
             JWT_APP_ID="${{ secrets.AKO_RELEASER_APP_ID }}" \
             VERSION="${{ steps.tag.outputs.version }}"
-      - name: Upgrade build Makefile & Dockerfile # TODO: remove after version 1.9 is deprecated
+      - name: Choose Dockerfile
+        id: pick-dockerfile
         run: |
-          git remote show origin
-          git checkout origin/${{ github.ref_name }} -- Makefile fast.Dockerfile scripts/version-check.sh scripts/module-path.sh
+          if test -f "fast.Dockerfile"; then
+            echo "dockerfile=fast.Dockerfile" >> $GITHUB_OUTPUT
+          else
+            echo "dockerfile=Dockerfile" >> $GITHUB_OUTPUT
```

Comment:
same question as above. Additionally, this looks like copy-pasta 🤔 can we consolidate this in a script?

Reply:
Same reply as above.

I also do not see an easy way to consolidate, note the previous step runs conditionally.